apps/wakunode2/wakunode2_validator_signed.nim

```diff
@@ -4,16 +4,20 @@ else:
   {.push raises: [].}
 
 import
+  std/[math,times],
```

Comment:
I think we should try to avoid importing `std/times` (especially when we use `chronos.timer` constructs in the same module). In this case we need it for absolute system time, but `waku_core/time` provides the wrappers for that.

Reply:
fixed 3092a519137b5db1663a479d6562da6ecfe98cac